packages/select/src/components/select/suggest.tsx

```diff
@@ -38,6 +38,12 @@ export interface ISuggestProps<T> extends IListItemsProps<T> {
     /** Custom renderer to transform an item into a string for the input value. */
     inputValueRenderer: (item: T) => string;
 
+    /**
+     * Optionally, if provided the selected item will be in controlled mode.
+     * Use the onItemSelect function to monitor changes.
```

Comment:
please edit this to match `activeItem` docs (including backticks around prop name): 

> The currently selected item, or null to indicate that no item is selected. If omitted, this prop will be uncontrolled (managed by the component's state). Use `onItemSelect` to listen for updates.